app/controllers/application.js

```diff
@@ -206,6 +206,18 @@ export default class ApplicationController extends ParachuteController {
   @action
   @trackEvent('click', 'popup', 'popupLocation')
   handleMapClick(e) {
+    const citymapLayerDisabled = this.get('model').layerGroups.toArray().filter((layerGroup) => { // eslint-disable-line
+      return layerGroup.get('visible');
```

Comment:
If you change this to `.filter(layerGroup => layerGroup.get('visible'))` you shouldn't need to disable eslint here.